### DISCUSSION TITLE
Can I have separate filesystem tabs?

### DISCUSSION THREAD

**Discussion by igrikus** [nvim-neo-tree/neo-tree.nvim#1425] [Q&A]
I want to have a tab with the current open project, then click `>` and switch to my `~/drafts` folder where I have work-in-progress stored

Neo-tree has a "source" concept, but I haven't found how to set up multiple tabs with the same source (in this case _filesystem_)

This is my usual workflow from JetBrains IDE where they have a concept of "scratches" that can be conveniently opened.

**Comment by pysan3** [comment]
I don't know if you understand what tab means in vim but you can open a separate neo-tree instance (which means you can open different cwd) for each separate vim tabpage. 

Other option would be to use `position = current`. You can open as many neo-trees as you want and all are independent.

**Comment by igrikus** [comment]
Hello @pysan3,thanks for such a quick response!

By tabs I meant "sources", sorry if I didn't explain it properly
I have the option to switch between them by pressing `>` (default mapping). I can view the currently open project (**cwd**), my git status, open buffers and so on
But I would like to have two sources with `filesystem` type. One for my current open project, one for my drafts.

I could open another tab (a real tab) as you suggest, but I usually have tabs with real projects open, and this is more of a "quick look at some file" concept

If there is such an option, that would be great! If not, I'll have to use workarounds and wean myself off old habits to preempt Alzheimer's 😄
Thanks in advance for your reply
